static/app/views/dashboardsV2/widgetBuilder/buildSteps/groupByStep/releaseGroupBySelector.tsx

```diff
@@ -0,0 +1,33 @@
+import {QueryFieldValue} from 'sentry/utils/discover/fields';
+import {useMetricsContext} from 'sentry/utils/useMetricsContext';
+import {FieldValueKind} from 'sentry/views/eventsV2/table/types';
+
+import {GroupBySelector} from './groupBySelector';
+
+interface Props {
+  columns: QueryFieldValue[];
+  onChange: (newFields: QueryFieldValue[]) => void;
+}
+
+export function ReleaseGroupBySelector({columns, onChange}: Props) {
+  const {metricTags} = useMetricsContext();
+
+  const tagKeys = Object.values(metricTags)
```

Comment:
Can you add a comment here on how this should reflect in the UI? if I understand correctly, we are taking the key of each tag and sorting it alphabetically (why by key though)?

Reply:
it was by keys, because of how this is structured

```
export type MetricsTag = {
  key: string;
};
```